### ISSUE TITLE
Mac arm64 release (and possibly also x64 release) not notarized and cannot be opened

### ISSUE THREAD

**Issue by rly** [NeurodataWithoutBorders/nwb-guide#630]
### Describe the issue

After downloading the latest NWB-GUIDE-arm64.dmg file, running it, dragging the NWB GUIDE icon to Applications, and then running the NWB GUIDE app from Applications, I get this error:
<img width="260" alt="Screenshot 2024-02-28 at 11 11 05 PM" src="https://github.com/NeurodataWithoutBorders/nwb-guide/assets/310197/150ce2e3-6f4b-4981-8be2-2e15bfc39e6f">

The latest Mac release build passed but appears to have had an error during notarization.

```
SyntaxError: Unexpected token E in JSON at position 0
    at JSON.parse (<anonymous>)
    at /Users/runner/work/nwb-guide/nwb-guide/node_modules/@electron/notarize/src/notarytool.ts:79:25
    at Generator.next (<anonymous>)
    at fulfilled (/Users/runner/work/nwb-guide/nwb-guide/node_modules/@electron/notarize/lib/notarytool.js:28:58)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

I will investigate now but could use some help

### Steps to Reproduce

See above.

### Operating System

Mac M1

### GUIDE Version

0.0.14

### Code of Conduct

Yes

### Did you confirm this issue was not already reported?

Yes

**Comment by rly** [comment]
My best guess is that I had not agreed to the latest Apple Developer Program License Agreement, and so when the CI tried notarize NWB GUIDE 0.0.14, the message `ERROR: You must first sign the relevant contracts online. (1048)` was returned ([ref](https://forums.developer.apple.com/forums/thread/117351)). This string does not parse as JSON - the error message starts with E. As a result, notarization fails.

I have since signed the latest Apple agreement, and so now I am re-running that release workflow to see if that fixed the issue. We'll see in an hour.

**Comment by rly** [comment]
Re-running the release workflow failed to upload/publish the new files to the existing 0.0.14 release because it can only publish to draft releases, so I converted the release to draft mode (which deleted that release and created a new one), deleted the assets, and re-ran both the windows and mac release workflows from the 0.0.14 release (so the release does not contain any commits after 0.0.14). The workflows added new assets to the draft release, and I published a new 0.0.14 release. Installing the `NWB-GUIDE-arm64.dmg` file from this release worked!
